packages/dart_frog_cli/README.md

```diff
@@ -1,5 +1,6 @@
-[![Dart Frog Logo][logo_white]][dart_frog_link_dark]
-[![Dart Frog Logo][logo_black]][dart_frog_link_light]
+[<img src="https://raw.githubusercontent.com/VeryGoodOpenSource/dart_frog/main/docs/static/img/dart_frog.png" align="left" />](https://dartfrog.vgv.dev/)
```

Comment:
I think we're missing the "Dart Frog" text being to the right of the logo now. In [Very Good CLI](https://github.com/VeryGoodOpenSource/very_good_cli/blob/main/README.md) is just plain text and it looks quite nice.